jwql/website/apps/jwql/urls.py

```diff
@@ -82,6 +82,7 @@
     path('query_submit/', views.query_submit, name='query_submit'),
     path('anomaly_query/', views.anomaly_query, name='anomaly_query'),
     path('table_viewer', views.jwqldb_table_viewer, name='table_viewer'),
+    path('table_viewer_monitor/<str:tablename_param>', views.jwqldb_table_viewer, name='table_viewer_monitor'),
```

Comment:
Naming this `table_viewer_monitor` is a bit misleading since this feature could concivably be used on any table within JWQLDB, not just monitor tables.  As such, I would suggest to rename the `table_viewer` URL/view (line 84) to something like `jwqldb/` (i.e. `https://dljwql.stsci.edu/jwqldb/`) and rename the `table_viewer_monitor` URL/view (line 85) to something like `jwqldb/<str:table_name>` (e.g. `https://dljwql.stsci.edu/jwqldb/nircam_bias_stats`)

Reply:
Hey @bourque I agree with you 100 percent. I had to add the second link because I don't know exactly how to handle the link in the navbar. The app will throw an error because it doesn't know how to handle this url.

https://github.com/spacetelescope/jwql/blob/dca658835404ba876430cf036fa8ce8304a1473c/jwql/website/apps/jwql/templates/base.html#L136